WeakAuras/Prototypes.lua

```diff
@@ -4768,6 +4768,31 @@ Private.event_prototypes = {
     automaticrequired = true,
   },
   -- BigWigs
+  ["BigWigs Stage"] = {
+    type = "addons",
+    events = {},
+    internal_events = {
+      "BigWigs_SetStage"
+    },
+    name = L["BigWigs Stage"],
+    init = function(trigger)
+      WeakAuras.RegisterBigWigsCallback("BigWigs_SetStage");
+      return [[
+         local stage = WeakAuras.GetBigWigsStage()
+      ]]
+    end,
+    args = {
+      {
+        name = "stage",
+        init = "arg",
+        display = L["Stage"],
+        type = "number",
+        conditionType = "number",
```

Comment:
Has this been tested? I think a store = true is missing to make conditions work.

Reply:
Yea, right now it seems to work fine. see: https://streamable.com/ui1mwu
I can try it with and without store = true and see what happens.

Reply:
I didn't see a difference in store with or without.